cleverhans/utils_mnist.py

```diff
@@ -11,7 +11,7 @@
 from . import utils
 
 
-def data_mnist():
+def data_mnist(testing=False):
```

Comment:
I'd suggest `train_start=0, train_end=60000, test_start=0, test_end=10000` for max flexibility.